src/lib/support/BUILD.gn

```diff
@@ -73,6 +73,42 @@ source_set("attributes") {
   ]
 }
 
+source_set("logging_header") {
+  sources = [ "logging/CHIPLogging.h" ]
+
+  public_deps = [
+    ":attributes",
+    ":logging_constants",
+    "${chip_root}/src/lib/core:chip_config_header",
+  ]
+
+  if (chip_pw_tokenizer_logging) {
+    public_deps += [ "${dir_pw_tokenizer}" ]
+  }
+}
+
+source_set("codeutils") {
+  # TODO: Why is ErrorStr.h not just in core along with CHIPError.h?
+  sources = [
+    "CodeUtils.h",
+    "ErrorStr.h",
```

Comment:
ErrorStr has a cpp file as well, they should go together I believe to ensure link order

Reply:
ErrorStr.cpp uses CHIPCore.h, which we can't depend on here.

Now in practice I think it just wants CHIPConfig.h, so I could make that change and move it into here...  Thoughts @andy31415 ?

Reply:
The ErrorStr bit is being addressed in https://github.com/project-chip/connectedhomeip/pull/29494